lib/modules/ens/index.js

```diff
@@ -60,7 +61,9 @@ class ENS {
     this.embark = embark;
     this.ensConfig = require('./ensContractConfigs');
     this.configured = false;
-
+    this.events.setCommandHandler('ens:configured', (cb) => {
```

Comment:
That's not what I meant. I meant, put `"ens:resolve"` here and in `this.ensResolve`, check if it is configured. That way, you can just call the resolve event from other modules and not have to verify it yourself.